assets/js/googlesitekit-widgets.js

```diff
@@ -19,7 +19,9 @@
 /**
  * Internal dependencies
  */
-import Widgets from './googlesitekit/widgets';
+import { Widget } from './googlesitekit/widgets';
+
+const Widgets = { componenets: Widget };
```

Comment:
This is not correct, it will override everything else that is currently in `Widgets`. This file shouldn't be modified at all, we should be adding `components.Widget` in `assets/js/googlesitekit/widgets/index.js`, where the rest of that object is defined.

Reply:
Arg, right, I must've got mixed up with how this was composed. Cheers!